README.rst

```diff
@@ -7,9 +7,60 @@ Janus Gateway RPM https://github.com/meetecho/janus-gateway
 
 Configuration
 --------------
-example configuration file are in /opt/janus/etc/janus
 
-Run
-----
+Example configuration files are in ``/opt/janus/etc/janus``
 
-Package install Systemd unit. Launc ```systemctl start janus-gateway```
+Install and start the service
+-----------------------------
+
+The RPM installs a systemd unit. To enable it and start the service run :: 
+
+    # systemctl enable --now janus-gateway
+
+The service *is not* restarted automatically after RPM upgrades. Remember to run ::
+
+    # systemctl restart janus-gateway
+
+
+Detailed trace
+--------------
+
+It is possible to build the RPM with additional libraries that send to standard output a detailed trace
+and detect memory leaks and faults. This is useful for upstream developers to debug a server crash.
+
+Run ``rpmbuild`` defining additional compilation flags. For instance: ::
+
+    $ rpmbuild -D 'dist .ns7' -D 'dbgflags -O0 -fno-omit-frame-pointer -g3 -ggdb3 -fsanitize=address,undefined' janus-gateway.spec
+
+The build requires a modern GCC version with libasan and libubsan dependencies. On CentOS 7 it is possible to install it from SCLo.
+
+See also the ``.travis.yml`` in this repository for more information.
+
+Once the RPM with additional libraries is installed:
+
+1. Add a local systemd unit override fragment, by copying the following 
+   contents to ``/etc/systemd/system/janus-gateway.service.d/override.conf``
+   or run ``systemctl edit janus-gateway.service``. ::
+
+    [Service]
+    ExecStart=
+    ExecStart=/bin/bash -c '\
+    export SOFIA_DEBUG=9; \
+    logfile=/var/log/janus-dbg-$(/bin/date +%%s).log ; \
+    echo OUT/ERR sent to $$logfile ; \
+    exec /opt/janus/bin/janus -o -d 7 -L /dev/null &>$$logfile'
```

Comment:
This is still hard to read. I'd move the SOFIA_DEBUG to `Environment` and remove the variable log name, using a fixed name lime `/var/log/janus-dbg.log`.
Whit this change, there is no need also to have this line `echo OUT/ERR sent to $$logfile ;`

Reply:
Ok for SOFIA_DEBUG (it does not simplify the file structure IMO)

Separate log files are needed when the server crashes and systemd restarts the service automatically. I could use the Bash builtin `$$` that expands to the shell PID: it is enough to get a unique file name.

Reply:
IMO a single file is more manageable and you should easily find there all restarts.
So I still prefer one file, but feel free to keep the current configuration if upstream requests multiple log files for support tickets.